ide/editor.lib2/src/org/netbeans/modules/editor/lib2/highlighting/HighlightsList.java

```diff
@@ -157,12 +159,14 @@ public AttributeSet cutSameFont(Font defaultFont, int maxEndOffset, int wsEndOff
 
         // Extends beyond first highlight
         Font firstFont = ViewUtils.getFont(firstAttrs, defaultFont);
+        Object firstPrependText = usePrependText && firstAttrs != null ? firstAttrs.getAttribute("virtual-text-prepend") : null;
```

Comment:
Is `// NOI18N` missing?
Should introduce the constant somewhere? But I could be wrong.